login-workflow/src/screens/InviteRegistrationPager.tsx

```diff
@@ -233,15 +234,15 @@ export const InviteRegistrationPager: React.FC = () => {
             customDetails
                 .slice(1)
                 //@ts-ignore
-                .filter((item: ComponentType<AccountDetailsFormProps>) => item !== null)
+                .filter((item: ComponentType<CustomRegistrationForm> | null) => item !== null)
                 //@ts-ignore
```

Comment:
You might be able to remove this ts ignore now...if not add a comment to explain why we had to leave it in:

```
// @ts-ignore there won't be any nulls after we filter them
```